jvm/advanced/JVM Advanced Example.md

```diff
@@ -0,0 +1,9 @@
+# JVM Advanced Example
+
+## Background
+
+This experiment builds upon our [JVM Recipe](https://github.com/redskyops/redskyops-recipes/tree/master/jvm). It adds the `gc_collector` parameter in order to tune the JVM garbage collection algorithm.
+
+## Prerequisites
+
+In order to run this experiment you must have a paid version of Red Sky Ops. To upgrade visit our [pricing page](https://www.carbonrelay.com/pricing/).
```

Comment:
Since we are switching from v1alpha1 -> v1beta1 we may need to make note here that this is only supported on red sky 1.6+ ( I think that's when we added in v1beta1 ).

Reply:
I'll add it in the base readme 👍